mola_gtsam_factors/README.md

```
@@ -1,10 +1,194 @@
-# License
 Copyright (C) 2018-2026 Jose Luis Blanco <jlblanco@ual.es>, University of Almeria
 
-This package is released under the GNU GPL v3 license as open source, with the main 
-intention of being useful for research and evaluation purposes.
-Commercial licenses [available upon request](https://docs.mola-slam.org/latest/solutions.html).
+This package is released under the **GNU GPL v3** license as open source, primarily intended for research and evaluation purposes.
+Commercial licenses are [available upon request](https://docs.mola-slam.org/latest/solutions.html) for proprietary applications.
+
+## Support
+
+- **Documentation**: https://docs.mola-slam.org/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)